packages/app-runtime/src/multiAccount/data/LocalAccount.ts

```diff
@@ -1,14 +1,15 @@
 import { ISerializable, Serializable, serialize, type, validate } from "@js-soft/ts-serval";
-import { CoreAddress, CoreDate, CoreId, ICoreDate } from "@nmshd/core-types";
+import { CoreAddress, CoreDate, CoreId, ICoreAddress, ICoreDate, ICoreId } from "@nmshd/core-types";
 
 export interface ILocalAccount extends ISerializable {
-    id: CoreId;
-    address?: CoreAddress;
+    id: ICoreId;
+    address?: ICoreAddress;
     name: string;
     directory: string;
     order: number;
     lastAccessedAt?: ICoreDate;
     devicePushIdentifier?: string;
+    deletionDate?: ICoreDate;
```

Comment:
Why only the deletion date and not the status too?

Reply:
b/c we don't care. 

1. deletionDate set = in deletion
2. deletionDate not set = not in deletion